packages/tools/kadena-cli/src/tx/commands/txLocal.ts

```diff
@@ -0,0 +1,80 @@
+import type { ICommandResult } from '@kadena/client';
+import { createClient } from '@kadena/client';
+import type { ChainId } from '@kadena/types';
+import type { Command } from 'commander';
+import { z } from 'zod';
+import { createCommand } from '../../utils/createCommand.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+import { log } from '../../utils/logger.js';
+import { generateClientUrl } from '../utils/txHelpers.js';
+import { arbitraryCodeTemplate } from './templates/templates.js';
+import { createTransaction } from './txCreateTransaction.js';
+
+export const createTxLocalCommand: (program: Command, version: string) => void =
+  createCommand(
+    'local',
+    'Submit pact code as a local call',
+    [
+      globalOptions.networkOptional({ disableQuestion: true }),
+      globalOptions.chainIdOptional({ disableQuestion: true }),
+    ],
+    async (option, { values }) => {
+      const { network, networkConfig } = await option.network();
+      const { chainId } = await option.chainId();
+      log.debug('tx:local', { network, networkConfig, chainId, values });
+
+      const code = values[0];
+      const templateChainId = (chainId ?? '0') as ChainId;
+      const templateNetworkId = networkConfig?.networkId ?? 'testnet04';
+      const templateNetworkHost =
+        networkConfig?.networkHost ?? 'https://api.testnet.chainweb.com';
```

Comment:
Why is this here, its been a while, but if I recall correctly when none is give it automatically would fall back to this. Well at least it used to.

Reply:
Yeah I wasn't sure how to do this. The problem is I want the network flag to be optional and not prompted. So unless you pass `-n mainnet` as example, you won't have an `networkConfig` object. 

I can change it so it fetches one of the existing networks instead of hardcoding default values here. But then we need to discuss the logic of picking a network

Reply:
we can go for this. it should fall back to something, wtihout going through all those hassles